tasks/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- fail: msg="Installed ansible version {{ ansible_version.full }}, but ansible version >= 2.1.1.0 required"
+  when: "{{ ansible_version.full | version_compare('2.1.1.0', '<') }}"
```

Comment:
I think this only applies when using ansible-galaxy to manage roles.
We've run into sooooo much trouble with differing ansible versions that silently fail due to some version-specific bugs/subtle difference that I prefer to fail explicitly.